changelog/18718.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+core: Add vault.core.locked_users telemetry metric to emit information about total number of locked users.
```

Comment:
There should also be an update to telemetry.mdx to document the new metric.

Reply:
I added this change in the documentation PR https://github.com/hashicorp/vault/pull/18478